test/trust-proxy.test.js

```diff
@@ -65,17 +69,15 @@ test('trust proxy, not add properties to node req', (t) => {
     reply.code(200).send({ ip: req.ip, host: req.host })
   })
 
-  t.teardown(app.close.bind(app))
```

Comment:
we can use:

```
t.after(app.close.bind(app))
```

Reply:
Good call out went with 
```
t.after(() => app.close())
```

unfortunately `t.after(app.close.bind(app))` produced this error
```
test at test/trust-proxy.test.js:146:1
✖ trust proxy protocol (6.521042ms)
  Error [FastifyError]: Callback for 'close' hook is not a function. Received: 'object'
      at server.<computed> [as close] (/Users/dancastillo/repos/fastify/fastify/node_modules/avvio/boot.js:254:13)
      at TestHook.runInAsyncScope (node:async_hooks:211:14)
      at TestHook.run (node:internal/test_runner/test:934:25)
      at TestHook.run (node:internal/test_runner/test:1225:18)
      at TestHook.run (node:internal/util:543:20)
      at node:internal/test_runner/test:853:20
      at process.processTicksAndRejections (node:internal/process/task_queues:105:5)
      at async Test.runHook (node:internal/test_runner/test:851:7)
      at async after (node:internal/test_runner/test:893:9)
      at async Test.run (node:internal/test_runner/test:942:7) {
    code: 'AVV_ERR_CALLBACK_NOT_FN',
    statusCode: 500
  }

```